src/libclipper/include/clipper/containers.hpp

```diff
@@ -31,13 +32,14 @@ class ModelContainer {
 
   size_t get_batch_size(Deadline deadline);
   double get_average_throughput_per_millisecond();
-  void update_throughput(size_t batch_size, long total_latency);
+  void update_container_stats(size_t batch_size, long total_latency);
```

Comment:
After talking to dan, the plan is to revert the extraction of AppMetrics and make a separate BenchMetrics (or something) in bench_util. For now, the two measure seemingly identical things, but that can easily change with what we choose to (or not to) benchmark (say, prediction accuracy, which isn't captured in AppMetrics). 

The change to `update_throughput` sounds good to me.